prod_cockpit_demo/dom0.xml

```diff
@@ -10,5 +10,5 @@
     <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
     <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
     <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
-    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="0a94decea3bd2504590d1637eadff9d502c19ee2" />
```

Comment:
Do we need this change?
This change is removed in following commit "Dom0: shift dom0 manifest to dunfell":
https://github.com/xen-troops/manifests/pull/88/commits/a112a529fd6ff65bb4a00b0de6e3f6f64bd62183#diff-c2246cac11b492d8a68de8a6bdd86e083e164bab1d70999b0de5df9d55cb2f9dL13